docker/cenit_web/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM cenit:latest
+
+ENV SKIP_ASSETS_COMPILE=false
+ENV SKIP_DB_INITIALIZATION=true
+ENV SKIP_RABBIT_MQ=true
+
+RUN set -x; \
+   bundle exec rake assets:precompile

```

Comment:
If you keep environment variable SKIP_ASSETS_COMPILE you must to have too:
```Dockerfile
RUN if $SKIP_ASSETS_COMPILE; then set -x; else bundle exec rake assets:precompile ; fi
```

Reply:
> If you keep environment variable SKIP_ASSETS_COMPILE you must to have too:
> 
> ```dockerfile
> RUN if $SKIP_ASSETS_COMPILE; then set -x; else bundle exec rake assets:precompile ; fi
> ```

@TFAssKID yes you are right, I think is just remove SKIP_ASSETS_COMPILE